### PR TITLE
[12.0][MIG] l10n_be & l10n_be_invoice_bba

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Migration scripts for `l10n_be_invoice_bba` module.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/account/migrations/12.0.1.1/post-migration.py [hunk 1]
```diff
@@ -220,6 +220,42 @@ def set_default_taxes(env):
         })
 
 
+def fill_res_company_invoice_reference_type(cr):
```
Comment (pedrobaeza) [inline addons/account/migrations/12.0.1.1/post-migration.py]: Why this change here?

Comment (MiquelRForgeFlow) [reply addons/account/migrations/12.0.1.1/post-migration.py]: Well, I am not really sure of this long query and the other long in the l10n_be_invoice_bba. I was waiting your opinion. I can remove them.

Comment (pedrobaeza) [reply addons/account/migrations/12.0.1.1/post-migration.py]: But I want to know why is needed to be done in account. Rationale?

Comment (MiquelRForgeFlow) [reply addons/account/migrations/12.0.1.1/post-migration.py]: Ah, because this part is `account` stuff. We didn't saw when we migrated `account`, the relation between `reference_type` and `invoice_reference_type`.

Comment (pedrobaeza) [reply addons/account/migrations/12.0.1.1/post-migration.py]: Then it should be a different PR...

Comment (MiquelRForgeFlow) [reply addons/account/migrations/12.0.1.1/post-migration.py]: done in https://github.com/OCA/OpenUpgrade/pull/1941.

File: addons/l10n_be_invoice_bba/migrations/12.0.1.2/post-migration.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+def map_res_company_invoice_reference_type(cr):
+    openupgrade.logged_query(
+        cr, """
+        UPDATE res_company
+        SET invoice_reference_type = 'structured'
+        WHERE invoice_reference_type = 'bba'"""
+    )
+
+
+def map_res_company_l10n_be_structured_comm(cr):
```
Comment (MiquelRForgeFlow) [inline addons/l10n_be_invoice_bba/migrations/12.0.1.2/post-migration.py]: I am not sure of this

Comment (MiquelRForgeFlow) [reply addons/l10n_be_invoice_bba/migrations/12.0.1.2/post-migration.py]: Discarded because https://github.com/OCA/OpenUpgrade/pull/1941 was also discarded. Too narrow cases, not worth.

**Review by pedrobaeza — APPROVED** [review]
